wcfsetup/install/files/acp/database/update_com.woltlab.wcf_6.1.php

```diff
@@ -35,4 +36,10 @@
                 ->referencedColumns(['userID'])
                 ->onDelete('CASCADE'),
         ]),
+    PartialDatabaseTable::create('wcf1_message_embedded_object')
+        ->indices([
+            DatabaseTableIndex::create('id')
```

Comment:
Please change to name of the index to something less generic (maybe to someting like `messageEmbeddedObject`).